packages/block-library/src/classic/edit.js

```diff
@@ -141,6 +141,11 @@ export default class ClassicEdit extends Component {
 		editor.on(
 			'Paste Change input Undo Redo',
 			debounce( () => {
+				// We need this check because when we remove the editor (onUnmount)
+				// due to the usage of `debounce`, this callback is executed in
+				// another tick. This results in setting the content to empty.
+				if ( editor._isRemoved ) return;
```

Comment:
I personally like this style, but I'm surprised the linter didn't complain about this `if` statement with no curly braces:

```js
if ( A ) {
  return;
}
```

Reply:
It might have `allowSingleLine` set. I'll check the rules.

Reply:
It seems that allows the omission of curly braces when a block contains only one statement by default, if not explicitly set. I'm not sure yet though.

There are other similar cases in the codebase as well. So if this is a problem, we should check and fix the linter config/rules.

Reply:
Yeah, apparently it's not enforced in code, but for reference here are the guidelines:

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#blocks-and-curly-braces